metadata.php

```diff
@@ -12,15 +12,32 @@
  */
 $sMetadataVersion = '2.1';
 
+// method to toggle the visibility of the DOM elements for the module terms and conditions content
+$sToggleJs = file_get_contents(dirname(__FILE__) . '/out/html/toggle-terms-of-use-js.html');
```

Comment:
Please, give it a more descriptive name like "sToggleTerms" if we are able to change it.